presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -176,7 +178,7 @@ public static Slice toISO8601FromTimestamp(ConnectorSession session, @SqlType(St
     }
 
     @ScalarFunction("to_iso8601")
-    @SqlType(StandardTypes.VARCHAR)
+    @SqlType("varchar(29)")
```

Comment:
29 seems wrong with today's Presto implementation of to_iso8601:
```
presto:presto> select to_iso8601(TIMESTAMP '12345-01-01 12:34:56.789 America/Los_Angeles');
             _col0
--------------------------------
 12345-01-01T12:34:56.789-08:00
```
By the way the implementation is also wrong, it should say "+12345" instead of "12345" per https://en.wikipedia.org/wiki/ISO_8601#Years